forward_engineering/api.js

```diff
@@ -133,7 +134,7 @@ const getEntityData = (container, entityId) => {
 const convertJsonToAvro = (jsonSchema, schemaName) => {
 	jsonSchema = { ...jsonSchema, type: 'record' };
 	const avroSchema = {
-		...convertSchema(jsonSchema),
+		...convertSchema(jsonSchema, getCustomPropertiesKeywords(getEntityLevelConfig(), jsonSchema)),
```

Comment:
I pass the customProperties argument here, because we need to distinguish entityLevelConfig properties from the 'record' fieldLevelConfig properties